### PR TITLE
create-and-add-a-cart-button

### PR DESCRIPTION
**Pull request by HaroldFS**
- setup ui cart button
- added the setup to the viewDidLoad

### CODE REVIEW

File: swift/ios-shoppe-demo/Views/StoreViewController.swift [hunk 1]
```diff
@@ -39,6 +37,26 @@ class StoreViewController: UICollectionViewController, UICollectionViewDelegateF
...
+    func setupNavigationBar() {
+        let barCartButton = UIBarButtonItem(image: UIImage(named: "Cart"), style: .done, target: self, action: #selector(openCart))
+        let textAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
+
+        barCartButton.tintColor = .white
+
+        navigationController?.navigationBar.titleTextAttributes = textAttributes
+        navigationController?.navigationBar.barTintColor = .systemIndigo
+
+        navigationItem.title = "Shoppee"
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Views/StoreViewController.swift]: Typo `Shoppe`.  Patrick and I have these titled as `Reactshoppe` and `Angular Shoppe`.  For consistency, you might consider using `iOS Shoppe`.

File: swift/ios-shoppe-demo/Resources/Assets.xcassets/Cart.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Cart.png",
```
Comment (van-fs) [inline swift/ios-shoppe-demo/Resources/Assets.xcassets/Cart.imageset/Contents.json]: The icon looks a bit skewed.  What about grabbing the cart icon (for iOS) from https://material.io/resources/icons/?search=cart&icon=shopping_cart&style=baseline.  This is the same icon used in the Angular app.

**Review by van-fs — APPROVED** [review]
LGTM, see my comments about changing the aesthetics.

File: swift/ios-shoppe-demo/Resources/Assets.xcassets/Cart.imageset/Contents.json (hunk 2 above)
Comment (HaroldFS) [reply swift/ios-shoppe-demo/Resources/Assets.xcassets/Cart.imageset/Contents.json]: sounds good :)

**Pull request merged**